
[notice] A new release of pip is available: 24.2 -> 24.3.1
[notice] To update, run: python3 -m pip install --upgrade pip
wandb: Using wandb-core as the SDK backend. Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: takonoselidze (takonoselidze-charles-university). Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.18.3
wandb: Run data is saved locally in /scratch.ssd/takonoselidze/job_7227137.pbs-m1.metacentrum.cz/wandb/run-20241203_155029-5nk4usie
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run rosy-thunder-5
wandb: ‚≠êÔ∏è View project at https://wandb.ai/takonoselidze-charles-university/RPP%20ensemble%20train-vit_b_16%20target-resnet50%201%20iters%20
wandb: üöÄ View run at https://wandb.ai/takonoselidze-charles-university/RPP%20ensemble%20train-vit_b_16%20target-resnet50%201%20iters%20/runs/5nk4usie
Traceback (most recent call last):
  File "main.py", line 450, in <module>
    results = start_iteration(device, patch_size, discriminators, dataloader, classes, target_classes, checkpoint_dir, num_of_epochs, brightness_factor, color_transfer)
  File "main.py", line 355, in start_iteration
    results = evaluate_saved_generators(checkpoint_dir, fixed_noise, patch_size, dataloader, deployer, discriminators, device)
  File "main.py", line 185, in evaluate_saved_generators
    target_class_checkpoints = load_checkpoint_by_target_class(checkpoint_files, checkpoint_dir)
TypeError: load_checkpoint_by_target_class() takes 1 positional argument but 2 were given
